styles/styles.css

```
@@ -1,7 +1,290 @@
+}
+
+.subheader .list {
+  position: relative;
+  margin-right: 35px;
+}
+
+.subheader .dropdown {
+  display: none;
+  position: absolute;
+  border: solid #A4A4A4;
+  border-width: 1px;
+  border-radius: 4px;
+  background-color: #ffffff;
+  list-style-type: none;
+  padding: 10px;
+  margin-top: 5px;
+}
+
+.subheader .dropdown li {
+  margin-bottom: 5px;
+}
+
+.subheader a {
+  text-decoration: none;
+}
+
+.subheader .nav-menu li:hover .dropdown {
+  display: block;
+}
+
+/* ==== FOOTER ==== */
+
+footer {
+  background-color: #000000;
+  color: #ffffff;
+  padding: 20px;
+  text-align: center;
+}
+
+footer .top {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+footer .top i {
+  margin-right: 10px;
+  font-size: 24px;
+}
+
+footer .bottom {
+  border-top: 1px solid #444;
+  padding-top: 10px;
+}
+
+footer .logo-img {
+  height: 28px;
+  padding-right: 5px;
+  width: auto;
+}
+
+.sitemap {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 20px;
+  background-color: #000;
+  padding: 40px;
+  color: #fff;
+  text-align: center;
+}
+
+.sitemap .section {
+  margin-bottom: 20px;
+}
+
+.sitemap .section h3 {
+  margin-bottom: 10px;
+  font-size: 1.2em;
+  color: #fff;
+}
+
+.sitemap .section p,
+.sitemap .section ul {
+  margin: 5px 0;
+}
+
+.sitemap .section ul {
+  list-style: none;
+  padding: 0;
+}
+
+.sitemap .section ul li {
+  margin-bottom: 5px;
+}
+
+.sitemap .section ul li a,
+.sitemap .section a {
+  color: #ccc;
+  text-decoration: none;
+}
+
+.sitemap .section ul li a:hover {
+  color: #fff;
+}
+
+.sitemap .section form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.sitemap .section input[type="email"] {
+  margin-bottom: 10px;
+  padding: 5px;
+  border-color: #ffffff;
+  width: 180px;
+  align-self: center;
+  background-color: #000000;
+  border-width: 1px;
+}
+
+.sitemap .section button {
+  padding: 5px 10px;
+  align-self: center;
+  border-width: 1px;
+  background-color: #000000;
+  border-color: #ffffff;
+  color: #fff;
+  cursor: pointer;
+  width: 180px;
+}
+
+.sitemap .section a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.sitemap .section a:hover {
+  text-decoration: underline;
+}
+
+input[type="text"]:focus {
+  outline: none;
+  border: none;
+}
+
+@media screen and (min-width: 850px) {
+  header {
+    flex-direction: row;
+    justify-content: space-between;
+    text-align: left;
+  }
+
+  .subheader .nav-menu {
+    justify-content: left;
+  }
+
+  header .header-search {
+    padding-right: 0;
+    margin-bottom: 0;
+    margin-left: 0;
+  }
+
+
+  .header-search input {
+    margin-left: 32%;
+    width: 40%;
+  }
+
+  .subheader {
+    text-align: left;
+  }
+
+  .subheader .list {
+    position: relative;
+    margin-right: 20px;
+  }
+
+  footer .top {
+    flex-direction: row;
+  }
+
+  footer .left,
+  footer .center,
+  footer .right {
+    margin-bottom: 0;
+  }
+

```

Esta es una idea, para que los iconos no queden tan pegados 

```suggestion
footer .center, footer .right {
  display: flex;
  gap: 30px;
}
```